recipes/dropstackframe/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "dropstackframe" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: dropstackframe
```

Would go ahead and use the `name` variable from above here

```suggestion
  name: {{ name }}
```